chemprop/train/run_training.py

```diff
@@ -200,7 +200,12 @@ def run_training(args: TrainArgs,
         else:
             debug(f'Building model {model_idx}')
             model = MoleculeModel(args)
-
+            
+        # Optionally, overwrite weights:
+        if args.frzn_mpnn or (args.frzn_ffn_layers > 0):
```

Comment:
This doesn't include a check for if there's a checkpoint_frzn path. If that's not provided you'll hit an error when you try to load the checkpoint but it won't be an informative error. I think you need to include that path in this check or raise an informative error somewhere.